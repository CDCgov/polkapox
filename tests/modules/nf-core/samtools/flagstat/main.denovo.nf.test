nextflow_process {

    name "Test Process SAMTOOLS_FLAGSTAT"
    script "modules/nf-core/samtools/flagstat/main.denovo.nf"
    process "SAMTOOLS_FLAGSTAT"

    test("Samtools Flagstat Denovo") {

        when {
            params {
            }
            process {
                """
                // define inputs of the process here. Example:
                input[0] = [ [ meta: 'test_sub'], file("${projectDir}/tests/nf-test-data/test_sub.bam"), file("${projectDir}/tests/nf-test-data/test_sub.bam.bai") ]
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
