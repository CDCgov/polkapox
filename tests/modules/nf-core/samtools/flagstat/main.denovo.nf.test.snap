{
    "Samtools Flagstat Denovo": {
        "content": [
            {
                "0": [
                    
                ],
                "1": [
                    
                ],
                "flagstat": [
                    
                ],
                "versions": [
                    
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "24.10.4"
        },
        "timestamp": "2025-06-27T13:53:10.017688368"
    }
}